taxonomies/ingredients_processing.txt

```diff
@@ -887,17 +887,17 @@ wikidata:en:Q1470757
 
 #<en:pulp
 ca:sense polpa
```

Comment:
ca: should not be the first language, but it was wrong before

Reply:
We could add an error in taxonomy linter if we have en or xx translation and it is not the first language.

Reply:
Maybe as an option first, or maybe we could add some configuration flags in the taxonomy files, to list which languages can be the first languages, and which properties are supported.

Reply:
Ok we could add a warning instead of an error.